client/my-sites/themes/controller.jsx

```diff
@@ -24,6 +30,7 @@ function makeElement( ThemesComponent, Head, store, props ) {
 			canonicalUrl={ props.canonicalUrl }
 			image={ props.image }
 			tier={ props.tier || 'all' }>
+			<DocumentHead title={ BASE_TITLE } />
```

Comment:
I did have it set to `props.title` initially, but in the mobile viewport things looked a bit strange with "Themes - SITENAME.wordpress.com".  Based off the patterns elsewhere in calypso ([#](https://github.com/Automattic/wp-calypso/blob/master/client/me/controller.js#L43), [#](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/stats/controller.js#L132) ) I opted for the section name - but feel free to change it up!